docs/runtimes/load-from-hub.md

```diff
@@ -129,7 +129,7 @@ Partial list of parameters:
 
 - **func** &mdash; function object or spec/code url.
 - **name** &mdash; name of the function (under the project).
-- **kind** &mdash; runtime kind e.g. job, nuclio, spark, dask, mpijob. Default: job.
+- **kind** &mdash; runtime kind e.g. job, Nuclio, Spark, dask, mpijob. Default: job.
```

Comment:
- **kind** &mdash; runtime kind e.g. job, nuclio, spark, dask, mpijob. Default: job.

The job types are not capitalized

Reply:
@jillnogold nice catch! will fix

Reply:
@rokatyy huge amount of cleanup - great job!!